spec/fixtures/ModifiedFile.kt

```diff
@@ -69,7 +69,7 @@ class ModifiedFile {
             "Log info modified"
         )
         logInfo {
-            "Log info modified $var"
+            "TEST Log info modified $var"
         }
```

Comment:
<table data-meta="generated_by_">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/eManPrague/danger-logging_lint/blob/4c15a257becac7bd5151bbfcad5e99cdc6b70f91/spec/fixtures/ModifiedFile.kt#L73"></span>Does this log comply with security rules? Check <a href="https://gitlab.eman.cz/csob/csob-smartbanking-an/-/wikis/Security-rules-for-logging">Security rules</a>.</td>
    </tr>
  </tbody>
</table>